resources/rgbscripts/alternate.js

```diff
@@ -59,7 +59,7 @@ var testAlgo;
   colorPalette.makeSubArray = function(_index) {
     var _array = new Array();
     for (var i = 0; i < colorPalette.collection.length; i++) {
-      _array.push(colorPalette.collection[parseInt(i)][parseInt(_index)]);
+      _array.push(colorPalette.collection[parseInt(i, 10)][parseInt(_index, 10)]);
```

Comment:
Is radix mandatory?
According to W3C docs it's not
https://www.w3schools.com/jsref/jsref_parseint.asp
However this is ECMA...

Reply:
... Codacy thinks, it's unsafe not to fix this. The parseint function would otherwise autodetect the basis for the conversion. So it is not necessary, but a bit safer.